### PR TITLE
EZP-23284: adjusted textline to accept a numeric as input-value 

### PR DESCRIPTION
**Pull request by SKrimo**
> issue: https://jira.ez.no/browse/EZP-23284

Add a new StringProcessor for ezstring to cast the input REST field value as String.

### CODE REVIEW

**Comment by SKrimo** [comment]
@andrerom @yannickroger

File: eZ/Publish/Core/REST/Common/FieldTypeProcessor/StringProcessor.php [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * File containing the TextLineProcessor class.
```
Comment (crevillo) [inline eZ/Publish/Core/REST/Common/FieldTypeProcessor/StringProcessor.php]: StringProcessor maybe?

Comment (SKrimo) [inline eZ/Publish/Core/REST/Common/FieldTypeProcessor/StringProcessor.php]: Thanks @crevillo.

I started on the TextLine but the processor is based on the returned identifier by getFieldTypeIdentifier.

**Comment by andrerom** [comment]
Only thing from my side is that it seems your commits are using different author email then on github meaning you won't get attribution to your account.

If you want to fix it, do:
```bash
git rebase -i master
# in the interactive rebase dialog change all commits but the first one to "f" so they are all merged into one

# after the rebase lets fix the author
git commit --amend --author "Your name <your@email.com>"

# then force push to your PR branch
git push -f <repo> <branch>
```

**Comment by alongosz** [comment]
Merged in f79d915.

**Pull request closed without merging**